components/selection/selection-checkbox.js

```diff
@@ -0,0 +1,109 @@
+import '../inputs/input-checkbox.js';
+import { css, html, LitElement } from 'lit-element/lit-element.js';
+import { SkeletonMixin } from '../skeleton/skeleton-mixin.js';
+
+/**
+ * A checkbox for use in selection components such as lists and tables.
+ * @fires d2l-selection-change - Dispatched when the checked state changes
+ */
+class Checkbox extends SkeletonMixin(LitElement) {
+
+	static get properties() {
+		return {
+			/**
+			 * State of the checkbox
+			 */
+			checked: { type: Boolean },
```

Comment:
Wondering whether this should be named `selected` instead.

Reply:
Yeah I like `selected` 🤷 . Or... does this even need to be exposed to consumers? Just wondering if they ever really need to set this state, or if it should always be set by the selection mixin.

Reply:
Sounds good - I will name it selected.  

I don't have concrete use-case, but I think consumers will need the ability to define list items to be initially selected.  We could keep it private and make it public if/when that comes up.  I don't feel super strongly about it, other than it's nice knowing the subscribes are correctly notified in this scenario. If I didn't have this in the demo page during dev, I might have introduced a couple pitfalls that might have made supporting that later trickier/riskier.